electron/src/ipc-communication.ts

```diff
@@ -20,4 +25,30 @@ function addSelectDirHandler(window: BrowserWindow | null): void {
     }
 }
 
+function getImagePathsFromDir(window: BrowserWindow | null): void {
+    if (!window) {
+        console.error('Window was null!');
```

Comment:
You could add return statement here and remove else to reduce unnecessary code nesting

Reply:
fixed in efea7bfee2c5a186e7c0ac4d8a50565bf832e133

Reply:
Please change it to throw Error instead of return to make it more visible

Reply:
fixed in c82b9b8f1203b3c4571b2c61eaeed0dc7a094835